content/docs/authorization/deployment/_index.md

```diff
@@ -9,7 +9,7 @@ tags:
 ---
 
 {{% pageinfo color="primary" %}}
-The CSM Authorization RPM is no longer actively maintained or supported. It will be deprecated in CSM 2.0. It is highly recommended that you use CSM Authorization Helm deployment or CSM Operator going forward.
+The CSM Authorization RPM is no longer actively maintained or supported. It will be deprecated in CSM 1.10.0. It is highly recommended that you use CSM Authorization Helm deployment or CSM Operator going forward.
```

Comment:
Can this be changed to "The CSM Authorization RPM will be deprecated in a future release. It is highly recommended that you use CSM Authorization Helm deployment or CSM Operator going forward."?

This notice is mentioned in a few places that requires changes.

Reply:
@gallacher : Can we have this comment addressed?